### ISSUE TITLE
Similar Project

### ISSUE THREAD

**Issue by gmclelland** [robertpiira/ingrid#3]
Interesting project, this looks similar to http://www.stacklayout.com

Just thought I would let you know.  Who knows, maybe each framework can pick up some tricks from the other.

-Glenn

**Comment by robertpiira** [comment]
Thanks!

Totally, many of these different frameworks share the same kind of ideas. Looking into how they solve problems and how they approach things could generate new fantastic ways of doing some cool css layouts. 

I would like to get my hands on some forward thinking CSS3 flexible box model based grid systems. Seen any out there?

**Comment by gmclelland** [comment]
No, don't know of any CSS3 grids systems.  The ones on the top of my list right now are:
http://zengrids.com/
http://github.com/ericam/compass-susy-plugin/
http://foundation.zurb.com/
http://goldilocksapproach.com/

The problem with a CSS3 grid system is that you would need a shim to support older browsers, unless you are using it for a web app.

**Comment by robertpiira** [comment]
All good, thanks for the heads up :)
